### PR TITLE
[MRG] Remove not Windows compatible api

### PR DESCRIPTION
**Pull request by jeremiedbb**

### CODE REVIEW

**Comment by jeremiedbb** [comment]
Also expose original max threads

File: tests/test_threadpoolctl.py [hunk 1]
```diff
@@ -18,6 +17,12 @@ def should_skip_module(module):
     return module['internal_api'] == "openblas" and module['version'] is None
 
 
+def effective_openmp_num_threads(nthreads, max_threads):
```
Comment (ogrisel) [inline tests/test_threadpoolctl.py]: `effective_num_threads`?

File: tests/test_threadpoolctl.py [hunk 2]
```diff
@@ -230,20 +234,23 @@ def test_multiple_shipped_openblas():
 
 @pytest.mark.skipif(not scipy_available(),
                     reason="requires scipy")
```
Comment (ogrisel) [inline tests/test_threadpoolctl.py]: cosmetics: can fit on one line.

**Review by ogrisel — APPROVED** [review]
LGTM!

File: tests/test_threadpoolctl.py [hunk 3]
```diff
@@ -18,6 +17,12 @@ def should_skip_module(module):
     return module['internal_api'] == "openblas" and module['version'] is None
 
 
+def effective_num_threads(nthreads, max_threads):
+        if nthreads is None or nthreads > max_threads:
+            return max_threads
+        return nthreads
```
Comment (ogrisel) [inline tests/test_threadpoolctl.py]: ident?

**Pull request merged**